artemis/reporting/modules/domain_expiration_scanner/reporter.py

```diff
@@ -0,0 +1,45 @@
+import os
+from typing import Any, Dict, List
+
+from artemis.reporting.base.language import Language
+from artemis.reporting.base.report import Report
+from artemis.reporting.base.report_type import ReportType
+from artemis.reporting.base.reporter import Reporter
+from artemis.reporting.base.templating import ReportEmailTemplateFragment
+from artemis.reporting.utils import get_top_level_target
+
+
+class DomainExpirationScannerReporter(Reporter):
+    CLOSE_DOMAIN_EXPIRATION_DATE = ReportType("close_domain_expiration_date")
+
+    @staticmethod
+    def create_reports(task_result: Dict[str, Any], language: Language) -> List[Report]:
+        if task_result["headers"]["receiver"] != "domain_expiration_scanner":
+            return []
+
+        if not task_result["status"] == "INTERESTING":
+            return []
+
+        if not isinstance(task_result["result"], dict):
+            return []
+
+        additional_data = task_result["result"]
+
+        return [
+            Report(
+                top_level_target=get_top_level_target(task_result),
+                target=f"http://{task_result['payload']['domain']}",
```

Comment:
the report is about a domain, let's make the target a domain

Reply:
I think we should fix this (the way the project code works in this regard).

Reply:
what do you mean - what should be fixed?

Reply:
We cannot say we can have URL or domain as target if we cannot handle both cases. Both cases should work.

Reply:
does https://github.com/CERT-Polska/Artemis/pull/544/commits/a1eb8936cf394d99a41b90bc9ff30ab8b40ba7b3 solve the problem?

Reply:
We cannot say we can have URL or domain as target if we cannot handle both cases. Both cases should work.

Reply:
you didn't respond whether a1eb893 solves the problem ;)

Reply:
Workaround is still a workaround even if it works and is still worth fixing.